sabre_smart_permission_examples/white_list_permission/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 Cargill Incorporated
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[package]
+name = "white_list_permission"
+version = "0.1.0"
+authors = ["Cargill, Incorporated"]
+
+[dependencies]
+protobuf = "2"
+sabre-sdk = "0.1.3"
+grid-sdk = { git = "https://github.com/RyanLassigBanks/grid", branch = "remove-pike-example-directory" }
```

Comment:
This should point at the actual grid-sdk

Reply:
Agreed. Currently master is broken, so I can't add it as dependency and have it compile, which seems bad. Once [pull 38](https://github.com/hyperledger/grid/pull/38) I will update.

Reply:
nvm I'm just gonna update it, but `white_list_permission` won't compile until, the grid-sdk is fixed.